app/repositories/apps.py

```diff
@@ -28,3 +39,52 @@ async def get_app_status_sub():
         yield json.dumps(app_list[i - 1])
         await asyncio.sleep(4)
         switch = not switch
+
+
+async def install_app_sub(app_id: str):
+    if(not app_id in available_app_ids):
+        raise HTTPException(
+            status.HTTP_400_BAD_REQUEST, detail="script does not exist"
+        )
+
```

Comment:
The error message could be a little more descriptive. To a user who doesn't know Blitz internals, this could be quite confusing. 

`detail="app ID not found"`